R/lint.R

```diff
@@ -410,16 +410,17 @@ Lint <- function(filename, line_number = 1L, column_number = 1L, # nolint: objec
   }
   max_col <- max(nchar(line) + 1L, 1L, na.rm = TRUE)
   if (!is_number(column_number) || column_number < 0L || column_number > max_col) {
-    cli_abort(c(
-      i = "{.arg column_number} must be an integer between {.val {0}} and {.code nchar(line) + 1} ({.val {max_col}})",
-      x = "Instead, it was {.val {column_number}}."
-    ))
+    cli_abort(
+      "{.arg column_number} must be an integer between \\
```

Comment:
please don't use the end-of-line `\\`, I find the behavior pretty confusing & it's not a well-documented part of the R parser.

Reply:
Note that this repo uses line width=120, so you also  don't need to squeeze your code so tightly :)

Reply:
@MichaelChirico Out of curiosity, what's the kosher way of doing readable, multiline strings? That is, one that doesn't use `\\`? 

Because this is what I do as well 😅

Reply:
I ended up not putting `\\` but still keeping line break. cli doesn't care about line breaks (or double spaces) like glue does. With cli, you have to put `f` to force a line break.

Lmk if you want another readjustment!

Reply:
> Out of curiosity, what's the kosher way of doing readable, multiline strings?

`paste()`, `paste0()`